Gaming The Syllabus/Assets/XML/Enemies/Goblin.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <CombatUnit xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Goblin" maximumHealth="20" 
-    dialogueColor="orange" currentHealth="15">
+    dialogueColor="orange" currentHealth="15" sprite="test">
```

Comment:
ahh ok I now see how sprites are utilized in the existing XML files. Great work on this!